pkg/converter/comments.go

```diff
@@ -4,7 +4,7 @@ import (
 	"strconv"
 	"strings"
 
-	"github.com/golang/protobuf/protoc-gen-go/descriptor"
+	"google.golang.org/protobuf/types/descriptorpb"
```

Comment:
```suggestion
	descriptor "google.golang.org/protobuf/types/descriptorpb"
```

If the intended style is to name the imports (without the `pb` suffix), happy to make that update. Which would also minimize the diff.

Reply:
I went ahead and did it in https://github.com/GoogleCloudPlatform/protoc-gen-bq-schema/pull/45/commits/f855e58f9192f2d45007c53546822f47dc20a670 for consistency.